tests/integration_tests/test_upgrade.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import pytest
+from pathlib import Path
+
+from tests.integration_tests.clouds import ImageSpecification, IntegrationCloud
+from tests.integration_tests.conftest import (
+    get_validated_source,
+    session_start_time,
+)
+
+log = logging.getLogger('integration_testing')
+
+
+def _output_to_compare(instance, file_path, netcfg_path):
+    commands = [
+        'hostname',
+        'dpkg-query --show cloud-init',
+        'cat /run/cloud-init/result.json',
+        'grep Trace /var/log/cloud-init.log',
+        'systemd-analyze',
+        'systemd-analyze blame',
+        'cloud-init analyze show',
+        'cloud-init analyze blame',
+        'cat {}'.format(netcfg_path),
+        'cloud-id'
+    ]
+    with file_path.open('w') as f:
+        for command in commands:
+            f.write('===== {} ====='.format(command) + '\n')
+            f.write(instance.execute(command) + '\n')
+
+
+@pytest.mark.sru_2020_11
```

Comment:
Is this mark appropriate here? Not sure if we're marking all tests we're including in an SRU, or only marking tests that are *only* relevant to that SRU.